modules/assim.sequential/R/sda.enkf.R

```diff
@@ -3,49 +3,56 @@
 ##' @author Michael Dietze and Ann Raiho \email{dietze@@bu.edu}
 ##' 
 ##' @param settings    PEcAn settings object
-##' @param IC          data.frame of initial condition sample (nens X nstate)
-##' @param prior       data.frame of model parameter sample (nens X nstate)
 ##' @param obs.mean    data.frame of observations of the mean of variables (time X nstate)
 ##' @param obs.cov      data.frame of observations of the sd of variables (time X nstate)
-##' @param processvar  flag for if process variance should be estimated or not
-##' @param sample.parameters flag for if parameters should come from meta.analysis or be defaults. if false parameters are set to defaults
+##' @param pick.trait.params  character vector of traits to sample if sample.parameters == TRUE
+##' @param given.process.variance flag for if parameters should come from meta.analysis or be defaults. if false parameters are set to defaults
```

Comment:
1. because for linkages you can't do all of the parameters. you have to pick a subset. it is in the settings but not in the sda part in the set up. I can move it over.
2. given.process.variance is supposed to be from DA that you did before not from the meta analysis.